src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskLifecycleTest.java

```diff
@@ -78,7 +78,7 @@ protected CachedConnectionProvider connectionProvider(
 
     // Should request a connection, then should close it on stop()
     EasyMock.expect(mockCachedConnectionProvider.getConnection()).andReturn(db.getConnection()).anyTimes();
-    mockCachedConnectionProvider.close();
+    mockCachedConnectionProvider.close(true);
```

Comment:
@mukkachaitanya what does passing "true" mean?

Reply:
True implies that the connector is stopping and we want to close the ConnectionProvider entirely. False means that the connector is closing the connection when it faces some transient errors. This was introduced as part of https://github.com/confluentinc/kafka-connect-jdbc/pull/1284